src/App.js

```diff
@@ -1,13 +1,60 @@
 import React from 'react';
+import Todo from './components/TodoComponents/Todo'
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: [],
+      task: '',
+      uid: '',
+      completed: false,
```

Comment:
Hey, here is something you are going to love.

`yarn add uuid`
`import uuidv4 from 'uudi'`
uuidv4() in the place of Date.now.... 

test it out its very nice :)